android/src/main/java/com/reactnativecommunity/imageeditor/ImageEditorModuleImpl.kt

```diff
@@ -296,30 +340,119 @@ class ImageEditorModuleImpl(private val reactContext: ReactApplicationContext) {
+                ExifInterface.TAG_FOCAL_PLANE_Y_RESOLUTION,
+                ExifInterface.TAG_PHOTOMETRIC_INTERPRETATION,
+                ExifInterface.TAG_PLANAR_CONFIGURATION,
+                ExifInterface.TAG_F_NUMBER,
+                ExifInterface.TAG_GAIN_CONTROL,
+                ExifInterface.TAG_GAMMA,
                 ExifInterface.TAG_GPS_ALTITUDE,
                 ExifInterface.TAG_GPS_ALTITUDE_REF,
+                ExifInterface.TAG_GPS_AREA_INFORMATION,
                 ExifInterface.TAG_GPS_DATESTAMP,
+                ExifInterface.TAG_GPS_DOP,
                 ExifInterface.TAG_GPS_LATITUDE,
                 ExifInterface.TAG_GPS_LATITUDE_REF,
                 ExifInterface.TAG_GPS_LONGITUDE,
                 ExifInterface.TAG_GPS_LONGITUDE_REF,
+                ExifInterface.TAG_GPS_STATUS,
+                ExifInterface.TAG_GPS_DEST_BEARING,
+                ExifInterface.TAG_GPS_DEST_BEARING_REF,
+                ExifInterface.TAG_GPS_DEST_DISTANCE,
+                ExifInterface.TAG_GPS_DEST_DISTANCE_REF,
+                ExifInterface.TAG_GPS_DEST_LATITUDE,
+                ExifInterface.TAG_GPS_DEST_LATITUDE_REF,
+                ExifInterface.TAG_GPS_DEST_LONGITUDE,
+                ExifInterface.TAG_GPS_DEST_LONGITUDE_REF,
+                ExifInterface.TAG_GPS_DIFFERENTIAL,
+                ExifInterface.TAG_GPS_IMG_DIRECTION,
+                ExifInterface.TAG_GPS_IMG_DIRECTION_REF,
+                ExifInterface.TAG_GPS_MAP_DATUM,
+                ExifInterface.TAG_GPS_MEASURE_MODE,
                 ExifInterface.TAG_GPS_PROCESSING_METHOD,
+                ExifInterface.TAG_GPS_SATELLITES,
+                ExifInterface.TAG_GPS_SPEED,
+                ExifInterface.TAG_GPS_SPEED_REF,
+                ExifInterface.TAG_GPS_STATUS,
                 ExifInterface.TAG_GPS_TIMESTAMP,
-                ExifInterface.TAG_IMAGE_LENGTH,
-                ExifInterface.TAG_IMAGE_WIDTH,
+                ExifInterface.TAG_GPS_TRACK,
+                ExifInterface.TAG_GPS_TRACK_REF,
+                ExifInterface.TAG_GPS_VERSION_ID,
+                ExifInterface.TAG_IMAGE_DESCRIPTION,
+                ExifInterface.TAG_IMAGE_UNIQUE_ID,
+                ExifInterface.TAG_ISO_SPEED,
+                ExifInterface.TAG_PHOTOGRAPHIC_SENSITIVITY,
+                ExifInterface.TAG_JPEG_INTERCHANGE_FORMAT,
+                ExifInterface.TAG_JPEG_INTERCHANGE_FORMAT_LENGTH,
+                ExifInterface.TAG_LENS_MAKE,
+                ExifInterface.TAG_LENS_MODEL,
+                ExifInterface.TAG_LENS_SERIAL_NUMBER,
+                ExifInterface.TAG_LENS_SPECIFICATION,
+                ExifInterface.TAG_LIGHT_SOURCE,
                 ExifInterface.TAG_MAKE,
+                ExifInterface.TAG_MAKER_NOTE,
                 ExifInterface.TAG_MODEL,
-                ExifInterface.TAG_ORIENTATION,
-                ExifInterface.TAG_SUBSEC_TIME,
+                ExifInterface.TAG_ORIENTATION, // removed
```

Comment:
I don't get why there is `// removed` comment. Is that property removed or about to be removed?

Reply:
I forgot to remove this comment, was added for testing purpose